SleepNumberController_Driver.groovy

```diff
@@ -139,6 +143,88 @@ def updated() {
   poll()
 }
 
+def uninstalled() {
+   deleteChildDevice(getChildDNI("outlet"))
+   deleteChildDevice(getChildDNI("underbedlight"))
+}
+
+def manageChildren() {
+    switch (state.type) {
+        case "presence":
+            def outlet = getOutlet()
+            if (!outlet) {
+                outlet = addChildDevice('hubitat','Generic Component Switch', getChildDNI("outlet"),
+                                        [ label: getChildName("Outlet"),
+                                         componentName: "outlet",
```

Comment:
Maybe naive question (but I haven't dealt w/ composite devices before) - the original post on [this](https://community.hubitat.com/t/composite-devices-parent-child-devices/1925) seems to say properties are name and label but you have componentName and componentLabel.  Are these synonymous?

Reply:
You are right this is a carryover from SmartThings code - I took this from another driver I have that was built to be cross-compatible before SmartThingsa dropped the classic app.

So I think we can drop the component* tags.  I'll do some cleanup on this.